android/src/com/frostwire/android/gui/views/KeywordTagView.java

```diff
@@ -85,24 +90,33 @@ private void updateComponents() {
         if (isInEditMode()) {
             return;
         }
-        TextView inclusiveIndicatorTextView = (TextView) findViewById(R.id.view_keyword_tag_inclusive_indicator);
+        ImageButton inclusiveIndicatorImageView = (ImageButton) findViewById(R.id.view_keyword_tag_inclusive_indicator);
         TextView keywordTextView = (TextView) findViewById(R.id.view_keyword_tag_keyword);
         TextView countTextView = (TextView) findViewById(R.id.view_keyword_tag_count);
+        LinearLayout tagContainer = (LinearLayout) findViewById(R.id.view_keyword_tag_container);
+
         if (count == -1) {
             countTextView.setVisibility(View.GONE);
         }
-        TextView dismissTextView = (TextView) findViewById(R.id.view_keyword_tag_dismiss);
-        inclusiveIndicatorTextView.setText(keywordFilter.isInclusive() ? "+":"-");
+        ImageView dismissTextView = (ImageView) findViewById(R.id.view_keyword_tag_dismiss);
+        inclusiveIndicatorImageView.setImageDrawable(getResources().getDrawable(keywordFilter.isInclusive() ? R.drawable.filter_add:R.drawable.filter_minus));
         keywordTextView.setText(keywordFilter.getKeyword());
         countTextView.setText("(" + String.valueOf(count) + ")");
         if (dismissible) {
+            tagContainer.setBackground(getResources().getDrawable(R.drawable.keyword_tag_background_active));
+            keywordTextView.setTextColor(getResources().getColor(R.color.app_text_white));
+            inclusiveIndicatorImageView.setBackground(getResources().getDrawable(R.drawable.keyword_tag_inclusive_background));
+            inclusiveIndicatorImageView.setColorFilter(new PorterDuffColorFilter(Color.parseColor("#2e9ec7"), PorterDuff.Mode.SRC_IN));
```

Comment:
do you think this`PorterDuffColorFilter` will work on older devices @aldenml (android-25) ?